packages/application-generic/src/usecases/add-job/add-job.usecase.ts

```diff
@@ -128,7 +128,7 @@ export class AddJob {
       _userId: job._userId,
     };
 
-    await this.queueService.addToQueue(
+    await this.standardQueueService.addMinimalJob(
```

Comment:
Queues have 2 different adding methods. One to add full event information and other for the minimal information to help to alleviate pressure on In-memory. Having it isolated in the QueueBase service in one method would help future improvements if we finally are able to move to save only the jobId for our flows.